src/track.jsx

```diff
@@ -26,12 +26,15 @@ var getSlideClasses = (spec) => {
   } else {
     slickActive = (spec.currentSlide <= index) && (index < spec.currentSlide + spec.slidesToShow);
   }
-  return classnames({
+  var { slidesClass } = spec;
+  var classNames = classnames({
     'slick-slide': true,
     'slick-active': slickActive,
     'slick-center': slickCenter,
     'slick-cloned': slickCloned
   });
+
+  return slidesClass ? `${classNames} ${slidesClass}` : classNames;
 };
```

Comment:
@victor-castellanos  I want the api of react-slick to be close to api of [slick](http://kenwheeler.github.io/slick/)
Since slick api didn't have slidesClass option we have to find another way to solve this usecase.

one way would be add custom styles to .slick-slide class.